Formula/imagemagick.rb

```diff
@@ -65,9 +65,14 @@ def install
       --disable-silent-rules
       --enable-shared
       --enable-static
-      --with-modules
     ]
 
+    if build.without? "modules"
```

Comment:
You'll need to actually add the related `option "without-modules", "Build without modules"` and same with threads.

Can you share a little more on why it would be desirable to ship without modules/threads?